app/views/measures/_rules_of_origin.html.erb

```diff
@@ -62,5 +62,17 @@
         <% end %>
       </tbody>
     </table>
+
+    <% for fta_intro in rules_of_origin.map(&:fta_intro).map(&:presence).compact %>
+    <div class="rules-of-origin-fta">
+      <h3 class="govuk-heading-m">
+        Trading relationship with <%= country_name %>
+      </h3>
+
+      <div class="tariff-markdown">
+        <%= govspeak fta_intro %>
```

Comment:
I'm not actually familiar with govspeak but I'm guessing it translates markdown to html following some css rules?